src/gleam_stdlib.erl

```diff
@@ -351,13 +351,9 @@ inspect(Any) when is_integer(Any) ->
 inspect(Any) when is_float(Any) ->
     io_lib_format:fwrite_g(Any);
 inspect(Binary) when is_binary(Binary) ->
-    case gleam@bit_string:is_utf8(Binary) of
-        true ->
-            Pattern = [$"],
-            Replacement = [$\\, $\\, $"],
-            Escaped = re:replace(Binary, Pattern, Replacement, [{return, binary}, global]),
-            ["\"", Escaped, "\""];
-        false ->
+    case inspect_maybe_utf8_string(Binary, []) of
```

Comment:
It'll be faster and use less memory to append to a binary rather than make a list as the accumulator here. No need to reverse either
```suggestion
    case inspect_maybe_utf8_string(Binary, <<>>) of
```